src/decoder/image.rs

```diff
@@ -77,6 +61,39 @@ impl TileAttributes {
     }
 }
 
+#[derive(Debug)]
+#[non_exhaustive]
+/// ChunkInfo describes the properties of a chunk (either tile or strip).
+pub struct ChunkInfo {
```

Comment:
This is really `pub(crate)`, right?

Reply:
I don't see any `pub use ChunkInfo`, and [this module isn't marked `pub`](https://github.com/image-rs/image/blob/04052e64c9a94606efc8bd3d87d5f3e0f566774e/src/lib.rs#L257).  Taken together, the struct isn't actually publicly exposed.

Reply:
I don't see any pub use ChunkInfo, and [this module isn't marked pub](https://github.com/image-rs/image-tiff/blob/master/src/decoder/mod.rs#L19). Taken together, the struct isn't actually publicly exposed.

(I think this is a good thing; if were guaranteeing a stable API for this struct, then I'd want to iterate on the interface)